src/components/endringslogg/endringslogg.less

```diff
@@ -0,0 +1,206 @@
+@top-radius: 0.625rem;
+
+.endringslogg {
+    margin-left: auto;
+    position: relative;
+}
+.endringslogg-knapp {
+    float: right;
+}
+.collapse-header {
+    background-color: #3E3832;
+    color: white;
+    padding: 1rem;
+    border-top-left-radius: @top-radius;
+    border-top-right-radius: @top-radius;
+    width: -webkit-fill-available;
+    height: 3rem;
+    min-height: fit-content;
+    text-align: center;
+}
+
+.collapse-container {
+    padding-top: 1rem;
+    margin-top: 2.8rem;
+    right: -0.5rem;
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    max-width: 30rem;
+    width: 30rem;
+    z-index: 2;
+
+}
+
+.collapse-container-enter {
+    height: 0;
+    overflow: hidden;
+    box-shadow: none;
+  }
+
+.collapse-container-enter-active {
+    height: 40rem;
+    overflow: hidden;
+    transition: all 400ms ease-in;
+    box-shadow: none;
+}
+
+.collapse-container-exit {
+    overflow: hidden;
+    height: 40rem;
+    box-shadow: none;
+}
+
+.collapse-container-exit-active {
+    height: 0rem;
+    overflow: hidden;
+    transition: all 400ms ease-out;
+    box-shadow: none;
+}
+
+.content {
```

Comment:
Stor fare for at det kan oppstå navne-kollisjoner ved `content` og andre klassenavn man ofte bruker.
Ville vurdert å legge all stylingen innenfor ett scope/namespace, eller bruke BEM-navngivning på klassenavnene